packages/verify-env/src/verify.ts

```diff
@@ -0,0 +1,83 @@
+import fs from "fs";
+import path from "path";
+
+export function verify(srcDir: string = "src/") {
+  // Get all source files
+  // Extensions to look for
+  const extensions = [".ts", ".tsx", ".js", ".jsx"];
+  const files = getFilesWithExtensions(srcDir, extensions);
+
+  // Initialize an array to store the extracted environment variables
+  const envVars: string[] = [];
+
+  // Read each file and extract environment variables
+  files.forEach((filePath) => {
+    try {
+      const fileContent = fs.readFileSync(filePath, "utf-8");
+      envVars.push(...extractEnvVars(fileContent));
+    } catch (error) {
+      console.error(`Error reading file ${filePath}: ${error}`);
+    }
+  });
+
+  // Remove duplicates
+  const uniqueEnvVars = [...new Set(envVars)];
+
+  // Validate
+  const badVars = validateVariables(uniqueEnvVars);
+  if (badVars.length > 0) {
+    console.error(
+      "Error: Some environment variables are not set or are empty:"
+    );
+    if (badVars.length > 0) {
+      console.error(`\n${badVars.join("\n")}\n`);
+    }
+    process.exit(1);
+  }
+}
+
+const getFilesWithExtensions = (
```

Comment:
Can probably replace this with one of https://www.npmjs.com/search?q=glob (ideally whatever is already used by webpack). Less code = fewer things that can break

Reply:
true and fixed.